pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/_http.log

```diff
@@ -278,11 +278,19 @@ X-Xss-Protection: 0
 GET https://cloudkms.googleapis.com/v1/projects/${projectId}/locations/us-central1/keyRings/kmskeyring-${uniqueId}/cryptoKeys/kmscryptokey-${uniqueId}:getIamPolicy?alt=json&options.requestedPolicyVersion=3&prettyPrint=false
```

Comment:
Also do we want to check in all of these _http.log files?

Reply:
I think yes to checking in the _http.log files, this is how we can verify what the controller is doing (e.g. is it actually making reasonable calls).  It also costs us basically nothing (a few files in a repo), to give us almost full coverage of the controller behaviour.

$projectId is the placeholder we insert in normalization, so that no matter what project we run this against (real or mock), the _http.log should be the same.  We use the same value as the input parameter because it feels natural, but we could also replace it with a different value if you think that's clearer!